contribution-acknowledgements.qmd

```diff
@@ -23,13 +23,15 @@ Roles in `DESCRIPTION` can be chosen from any of those listed in the [MARC Code
 > - **ths**: If the package is part of a thesis, use for the thesis advisor.
 > - **trl**: If the R code is merely a translation from another language (typically S), use for the translator to R.
 
+We include the ORCID of authors, where available, as part of their inclusion in the `DESCRIPTION`. This helps disambiguate in case of a common name and ensures that authors are properly credited.
```

Comment:
See also https://github.com/epiverse-trace/blueprints/pull/129 which is addressing the same point